systemtest/src/main/java/io/strimzi/systemtest/utils/specific/CruiseControlUtils.java

```diff
@@ -29,8 +32,37 @@
     private static final String CRUISE_CONTROL_MODEL_TRAINING_SAMPLES_TOPIC = "strimzi.cruisecontrol.modeltrainingsamples"; // partitions 32 , rf - 2
     private static final String CRUISE_CONTROL_PARTITION_METRICS_SAMPLES_TOPIC = "strimzi.cruisecontrol.partitionmetricsamples"; // partitions 32 , rf - 2
 
+    private static final int CRUISE_CONTROL_DEFAULT_PORT = 9090;
+    private static final String CRUISE_CONTROL_ENDPOINT = "/kafkacruisecontrol/";
```

Comment:
it's not an endpoint on its own, should we call it `CRUISE_CONTROL_BASE_ENDPOINT` or maybe better `CRUISE_CONTROL_ENDPOINT_BASE`? @tomncooper wdyt? Or using `ROOT` instead of `BASE` ...

Reply:
Yeah all the endpoints start with this and it is not endpoint on its own AFAIK